sidebar.json

```diff
@@ -13,7 +13,8 @@
   ],
   "Interfaces": [
     { "title": "HTTP", "href": "/docs/http/" },
-    { "title": "Schedule Tasks", "href": "/docs/scheduled-tasks/" },
+    { "title": "Schedules", "href": "/docs/scheduled-tasks/" },
```

Comment:
Let me know if you're ok with the change from "Schedule Tasks" to "Schedules".

Reply:
Yeah, I think this has been a subtask for the dashboard as well. Tasks should be in a different realm than Schedules.